sycl/source/detail/device_info.hpp

```diff
@@ -493,6 +493,23 @@ template <> struct get_device_info<device, info::device::parent_device> {
   }
 };
 
+// Specialization for bfloat16
+template <>
+struct get_device_info<bool, info::device::ext_intel_bf16_conversion> {
+  static bool get(RT::PiDevice dev, const plugin &Plugin) {
+    // TODO: We claim, that all Intel GPU devices support bfloat16 conversion
+    // feature but we'd better have a low-level extension to query for support
+    // of the feature
```

Comment:
Is there any spec for the extension?

Reply:
Here it is https://github.com/intel/llvm/pull/4237

Reply:
It seems #4237 is a spec for SYCL extension. Is there any spec for "a low-level extension"?

Could you please clarify why the temporary solution thru checking platform name is needed? Can we wait and implement the proper solution?

Reply:
Coping my answer from below.

>> It seems #4237 is a spec for SYCL extension. Is there any spec for "a low-level extension"?

Nothing to share right now

>> Could you please clarify why the temporary solution thru checking platform name is needed? Can we wait and implement the proper solution?

So users could already write bfloat16 - sensitive code using aspect traits deciding for them which version of kernel should be enqueued: with or without bf16 conversion feature.